app/shopcart.py

```diff
@@ -385,6 +470,67 @@ def create_shopcarts():
 @app.route('/shopcarts/<int:sid>/products', methods=['POST'])
 def create_products(sid):
 
+    """
+    Creates a Product
+    This endpoint will create a Product based the data in the body that is posted
+    ---
+    tags:
+      - Shopcarts
+    consumes:
+      - application/json
+    produces:
+      - application/json
+    parameters:
+      - name: sid
+        in: path
+        description: the sid of the shopcart to which you want to add products
+        required: true
+        type: integer
+      - in: body
+        name: body
+        required: true
+        schema:
```

Comment:
This is not the same as we used in the method body, and Flasgger 'Try it out' won't work.